src/creature.cpp

```diff
@@ -1425,9 +1425,9 @@ units::mass Creature::weight_capacity() const
 /*
  * Drawing-related functions
  */
-void Creature::draw( const catacurses::window &w, int player_x, int player_y, bool inverted ) const
+void Creature::draw( const catacurses::window &w, int x, int y, bool inverted ) const
```

Comment:
How about origin_x/origin_y both here and in the header?
```suggestion
void Creature::draw( const catacurses::window &w, int origin_x, int origin_y, bool inverted ) const
```